benchmark/tangurnis/notes.txt

```diff
@@ -1 +1,14 @@
 This directory contains benchmarks defined in Tan, Gurnis '07 to check compressible convection
+
+
+
+Comparison of the heating terms for TALA, Di=0.5, gamma=1, a=0:
+
+                                       shear heating  adiabatic_heating
+
+analytical solution (Tan, Gurnis '07)  5.306916e-03   5.251999e-03
+
+numerical solution (refinement 4)      5.30690178e-03 5.25196977e-03
+numerical solution (refinement 5)      5.30691533e-03 5.25199721e-03
```

Comment:
But also what an interesting testcase where you get the right answer to five digits already on the coarsest mesh. Maybe not the best chosen output quantity then...